### ISSUE TITLE
Can this library be published to maven central?

### ISSUE THREAD

**Issue by mahendrabishnoi2** [jacek99/structlog4j#9]
Since jcenter maven repository is now read-only, can you please publish this library to maven central. As this library is only available on jcenter, we are not able to remove jcenter repository from our build.
Reference: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

**Comment by jacek99** [comment]
Yes, I am actually working on it as part of a new 1.1.0 release.

**Comment by mahendrabishnoi2** [comment]
@jacek99 Let me know if I can help with publishing existing version to maven central.

**Comment by twokodes** [comment]
hello,

Please is this available on maven central?
